rust/main/rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.85"
+channel = "1.81.0"
```

Comment:
Why do we need to downgrade?

Reply:
This slows down SDK CI and kills the cache basically.

Reply:
But it worked with 1.88, isn't it? 

> of how fragile things are
So let's not bring this fragility into SDK via slow/flaky CI.

Not a blocker for this PR, I am fine to keep it closer to upstream

Reply:
> But it worked with 1.88, isn't it?

I'm not sure, its 1.81 in their repo: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/main/rust-toolchain

> So let's not bring this fragility into SDK via slow/flaky CI.

I strongly recommend we take a "if its not broken dont touch it approach" with hyperlane lol, espicially for upstreaming - I don't want to have debates with them over things like this